metacat-connector-hive/src/main/java/com/netflix/metacat/connector/hive/HiveConnectorDatabaseService.java

```diff
@@ -188,6 +189,10 @@ public boolean exists(@Nonnull @NonNull final ConnectorContext requestContext,
                 }
                 qualifiedNames.add(qualifiedName);
             }
+            //supporting sort by qualified name only
+            if (sort != null) {
+                ConnectorUtils.sort(qualifiedNames, sort, Comparator.comparing(QualifiedName::toString));
```

Comment:
You want to sort on the whole qualified name here or just the name of the database itself? I ask cause it's less efficient to turn the catalog and database name into a string and then do the comparison rather than just comparing on the database name only